scrapy_splash/middleware.py

```diff
@@ -45,8 +46,13 @@ class SplashCookiesMiddleware(object):
     It should process requests before SplashMiddleware, and process responses
     after SplashMiddleware.
     """
-    def __init__(self):
+    def __init__(self, debug=False):
         self.jars = defaultdict(CookieJar)
+        self.debug = debug
+
+    @classmethod
+    def from_crawler(cls, crawler):
+        return cls(debug=crawler.settings.getbool('COOKIES_DEBUG'))
```

Comment:
Yep, agreed! I tried to document them in c06da89e027c9f5e46836c81353360e3822c6165 - I'm not entirely sure about the ``SPLASH_LOG_400`` description.